csrc/python_frontend/fusion_record.h

```diff
@@ -1251,13 +1251,26 @@ struct TensorRecord : RecordFunctor {
       is_expand[index] = is_broadcast && has_non_broadcast_size;
     }
 
+    // skip stride_order if it's alloc_domain is in the same order as with root
+    // domain. We don't need this and we should be able to just use
+    // stride_order_, but currently alloc_domain support isn't ideal and could
```

Comment:
Can you point to the problematic code? Maybe some code is over nervous about allocation domain is specified at all rather than figure out it's actually trivial and has the same effect as an empty allocation domain?

Reply:
> I'll open an issue to track this when I get some better ideas.

Sure -- I'm fine with a TODO. Regardless whether you manage to root cause this, I'd open the issue first. This way, I can say `TODO(<issue id>)` here so readers have something to refer to :)